components/ButtonBar/src/MDCButtonBar.m

```diff
@@ -388,8 +407,15 @@ - (void)setItems:(NSArray<UIBarButtonItem *> *)items {
     }
 
     NSArray<NSString *> *keyPaths = @[
-      kEnabledSelector, NSStringFromSelector(@selector(title)),
-      NSStringFromSelector(@selector(image))
+      NSStringFromSelector(@selector(accessibilityHint)),
```

Comment:
Since these are used twice, please consider making them constants like `kEnabledSelector`

Reply:
kEnabledSelector was added as such because of compiler warnings - I'd prefer we keep referring to these APIs in a compile-time-safe way barring any definitive metrics that indicate we should do otherwise.